test/core/slice/b64_decode_fuzzer.cc

```diff
@@ -0,0 +1,31 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <fuzzer/FuzzedDataProvider.h>
```

Comment:
You don't actually need this header, we created our own system for that exact purpose. See the following files:

https://github.com/grpc/grpc/blob/22d8f8f0850f34af1b89ef71138efb84bfddd87e/test/core/util/one_corpus_entry_fuzzer.cc

https://github.com/grpc/grpc/blob/618a3f561d4a93f263cca23abad086ed8f4d5e86/test/core/util/grpc_fuzzer.bzl

https://github.com/grpc/grpc/blob/24b529e408bcf10f067663f1bb745b08a58f898c/test/core/json/BUILD#L23-L33

Reply:
And a usage of it here: https://github.com/grpc/grpc/blob/24b529e408bcf10f067663f1bb745b08a58f898c/test/core/json/fuzzer.cc

Reply:
Apologies, I misread the API and what you were using it for.

Still, we might be able to do without it. It's just yet another serializer, and we're all about that...

Reply:
Copying my comment from your other PR here:
Because FuzzedDataProvider doesn't work in a standalone build of gRPC (one without oss-fuzz), I suggest using a proto instead. I don't think we have a proto fuzzer, but you could define a proto with the relevant structure and construct it from this raw data. In the event the proto is not valid, you can just return 0. If it's valid, you can proceed with the actual fuzzing.

Reply:
Eh, proto feels overkill for this. I'm just going to roll my own implementation of ConsumeBool() :)